ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -135,6 +135,23 @@ class MySqlNamedBlobDb implements NamedBlobDb {
   private static final String SOFT_DELETE_QUERY_V2 =
       String.format("UPDATE %s SET %s = ? WHERE %s", NAMED_BLOBS_V2, DELETED_TS, PK_MATCH_VERSION);
 
+  /**
+   * Pull the stale blobs that need to be cleaned up
+   */
+  private static final String GET_STALE_QUERY = String.format(
+      "SELECT %s, %s, %s, %s, %s, %s FROM %s t1 LEFT JOIN (SELECT account_id, container_id, blob_name, max(version) as version FROM %s "
+          + "WHERE blob_state=1 GROUP BY account_id, container_id, blob_name) t2 "
+          + "ON (t1.account_id,t1.container_id,t1.blob_name,t1.version) = (t2.account_id,t2.container_id,t2.blob_name,t2.version) "
+          + "WHERE t1.gg_modi_ts<(CURRENT_TIMESTAMP(6) - INTERVAL 60 DAY) and t2.account_id IS NULL and t2.container_id IS NULL and t2.blob_name IS NULL and t2.version IS NULL",
+      ACCOUNT_ID,
+      CONTAINER_ID,
+      BLOB_NAME,
+      BLOB_ID,
+      VERSION,
+      CURRENT_TIME,
+      NAMED_BLOBS_V2,
+      NAMED_BLOBS_V2);
```

Comment:
Can you turn off the formatter and reformat this query statement just like list_query_v2?

Reply:
Sure, done.